CMakeLists.txt

```
@@ -20,6 +20,11 @@ option(C2A_USE_ALL_CORE_APPS        "Use C2A-core all Applications" ON)
 option(C2A_USE_ALL_CORE_TEST_APPS   "Use C2A-core all Test Applications" ON)
 option(C2A_USE_ALL_CORE_LIB         "Use C2A-core all library" ON)
 
+## C2A CCSDS select
+# See alse; /docs/core/ccsds.md
+option(C2A_USE_CCSDS_AOS_SPACE_DATA_LINK_PROTOCOL  "Use C2A-core CCSDS AOS SPACE DATA LINK PROTOCOL" OFF)
+option(C2A_USE_CCSDS_TC_SPACE_DATA_LINK_PROTOCOL   "Use C2A-core CCSDS TC SPACE DATA LINK PROTOCOL" OFF)
```

@sksat 名前これで良い？

C2A_USE_CORE_CCSDS ~~


のほうがいい？

---

めっちゃ長くなるけど一旦 `C2A_USE_CORE_CCSDS_*` かなあ．これが長いのは元の名前が長いせいなので......
あと，description の方は `~ implementation` とかの方がよさそう．単に use hoge protocol だと hoge protocol が勝手に使われるように思うけど，たぶんそうではないよね．

```suggestion
option(C2A_USE_CORE_CCSDS_AOS_SPACE_DATA_LINK_PROTOCOL  "Use C2A-core CCSDS AOS SPACE DATA LINK PROTOCOL implementation" OFF)
option(C2A_USE_CORE_CCSDS_TC_SPACE_DATA_LINK_PROTOCOL   "Use C2A-core CCSDS TC SPACE DATA LINK PROTOCOL implementation" OFF)
```

---

OK
あとでsedしておきます